plugins/catalog/src/js/repositories/components/RepositoriesDeleteConfirm.js

```diff
@@ -11,11 +12,11 @@ const RepositoriesDeleteConfirmMessage = ({ repository, error }) => {
 
   return (
     <div>
-      <p>
-        {`Repository (${label}) will be ${UserActions.DELETED}
-          from ${Config.productName}. You will not be able to install any
-          packages belonging to that repository anymore.`}
-      </p>
+      <Trans render="p">
+        Repository ({label}) will be {UserActions.DELETED}
+        from {Config.productName}. You will not be able to install any packages
```

Comment:
Does this render with a space between packages and belonging? I'm seeing mixed results in testing.